myhdl/conversion/_toVHDL.py

```diff
@@ -113,6 +113,7 @@ class _ToVHDLConvertor(object):
                  "use_clauses",
                  "architecture",
                  "std_logic_ports",
+                 "no_initial_value"
```

Comment:
@jck Do you mean `no_initial_values` or `initial_values`? I've no strong opinion so happy to take guidance.